frontend/luasettings.lua

```diff
@@ -110,14 +145,43 @@ function LuaSettings:flush()
     end
 end
 
+--- Closes settings file.
 function LuaSettings:close()
     self:flush()
 end
 
+--- Purges settings file.
 function LuaSettings:purge()
     if self.file then
         os.remove(self.file)
     end
 end
 
+--- Returns normalized version of KOReader git-rev input string.
+-- @string rev full KOReader git-rev such `v2015.11-982-g704d4238`
+-- @treturn int revision in the form of a number such as `201511982`
+-- @treturn string short git commit version hash such as `704d4238`
+function LuaSettings:normalized_version(rev)
```

Comment:
I feel like version related helper functions should live in utils.

We can also create a standalone reusable version module since `common_info_menu_table.lua` also reads and caches the version info.